### ISSUE TITLE
Antialiased circle/rectangle

### ISSUE THREAD

**Issue by madmaxio** [sotrh/learn-wgpu#375]
Would you mind adding example for drawing antialiasing circle/rectangle? It's interesting to see the approach.

**Comment by sotrh** [comment]
Are you talking about vector type graphics, or simply enabling multi-sampling?

**Comment by madmaxio** [comment]
I mean just drawing it with primitives, I found some examples from opengl/metal etc, they all say about smoothstep function.
So it is about using primitves + smoothstep.

**Comment by madmaxio** [comment]
Here is an example of what I'm talking about, absolutelly briliant article imho, but it is about metal. But the content is exactly the same - primitive basics.
https://www.warp.dev/blog/how-to-draw-styled-rectangles-using-the-gpu-and-metal

**Comment by sotrh** [comment]
Yeah I could probably do something like that with the signed distance field example I've been thinking of doing.

**Comment by madmaxio** [comment]
@sotrh I also need some wgpu related consulting on a payment basis, can you be up for this?

**Comment by sotrh** [comment]
@madmaxio what did you have in mind?

**Comment by madmaxio** [comment]
@sotrh I need to load and render a couple of gltf models with wgpu. So far I only succeeded with basic cube model. And also make a correct camera implementation.

**Comment by sotrh** [comment]
What do you mean by "correct camera implementation"? Also is there a better way I can contact you?

**Comment by madmaxio** [comment]
@sotrh Discord maximb 5928, or, drop your contacts.

**Comment by sotrh** [comment]
I sent you a friend request.
